south-african-journal-of-animal-science.csl

```diff
@@ -39,7 +39,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name delimiter=", " and="text" delimiter-precedes-last="always" initialize-with=". " name-as-sort-order="first"/>
+      <name delimiter=", " and="symbol" delimiter-precedes-last="always" initialize-with=". " name-as-sort-order="always"/>
```

Comment:
@Yusufras, `name-as-sort-order="always"` is not valid CSL. It should be `name-as-sort-order="all"` (see the description of "name-as-sort-order" under http://docs.citationstyles.org/en/stable/specification.html#name). Could you correct this?